sass/responsive/_mobile.scss

```diff
@@ -1851,6 +1851,11 @@
     .signup-team-all {
         max-height: calc(75vh - (118px + 187px));
     }
+    
+    .signup-team-all .infinite-scroll {
+        overflow-y: auto;
+    }
```

Comment:
This selector doesn't seem to hit the element I presume that it targets.

```html
<div class="infinite-scroll signup-team-all">
```
?
```css
.signup-team-all.infinite-scroll {
    overflow-y: auto;
}
```

digging into this further, though.. I don't know that this is even needed, as the following already kicks in on hover inside (e.g. you start to drag on mobile)
```css
.infinite-scroll:hover {
    overflow-y: scroll;
    overflow-y: overlay;
}
```

Reply:
yes, I have not checked in that way, I think it working on the hover, so I deleted this class now 
```js
.signup-team-all.infinite-scroll {
    overflow-y: auto;
}
```